src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { BasicRadioInput } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { TranslationUtils } from '../../../../utils';
+
+export const GenericRadioInput = ({ parameterData, parametersState, setParametersState, context }) => {
+  const { t } = useTranslation();
+  let enumValues = parameterData.enumValues;
```

Comment:
Starting with v5., enum values will no longer get retrieved from `parameterData.enumValues` but from `parameterData.options.enumValues` instead. You can use this instead:
`let enumValues = ConfigUtils.getParameterAttribute(parameterData, 'enumValues');`

See _src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericEnumInput.js_ for an example

Reply:
Should I do this modification in others generic inputs ?